IdentityCore/src/controllers/MSIDSilentController.m

```diff
@@ -123,7 +129,7 @@ - (void)acquireTokenWithRequest:(MSIDSilentTokenRequest *)request
             [telemetryEvent setIsExtendedLifeTimeToken:result.extendedLifeTimeToken ? MSID_TELEMETRY_VALUE_YES : MSID_TELEMETRY_VALUE_NO];
             if (self.isLocalFallbackMode)
             {
-                 [telemetryEvent setSsoExtFallBackFlow:1];
+                [telemetryEvent setSsoExtFallBackFlow:1];
```

Comment:
nit: revert back as unnecessary?

Reply:
I did a format code for the whole file to fix indentations :)